src/content/manifest/hybrid-development.mdx

```diff
@@ -20,6 +20,8 @@ If you want to adopt this approach, you need to keep in mind that developing wit
 ### How hybrid development works?
 Usually, okteto allows you to develop a service locally and run it in a production-like cloud environment synchronizing the local changes with the development container running in the cloud (sync mode) but okteto also allows you to use another approach in which the service to be developed runs locally and the rest of the application components run in the cloud (hybrid mode).
 
+Using *hybrid* mode a remote container is also raised because when a component of the cluster needs to consume the service being developed locally, the `reverse` is used to forward to the desired port of the development container what is exposed locally. In this way, for the rest of the cluster components the *hybrid* or *sync* mode is indifferent.
```

Comment:
Not sure if this paragraph is clear to me. Let's try to define it in a way that explains what is shown in the diagram.

Reply:
ok! take a look at this paragraph and let me know if it is ok https://github.com/okteto/docs/pull/557/files#diff-4704534dddb0360cc8e8c8559f984492563b94235123bf13714fd4867ad9933aR23